stl/inc/xatomic.h

```diff
@@ -10,6 +10,9 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #include <intrin0.h>
+#if 1 // TRANSITION, Visual Studio 2019 to include _mm_pause in <intrin0.h>

```

Comment:
I think it would be better to file an issue to track this change to `<intrin0.h>` and reference that issue in the TRANSITION comments here.